src/main/resources/application.yml

```diff
@@ -59,4 +50,12 @@ logging:
 
 springdoc:
   packagesToScan: com.iexec.sms
-  pathsToMatch: /**
\ No newline at end of file
+  pathsToMatch: /**
+
+tee:
```

Comment:
Can we move that block to `application-scone.yml`?

Reply:
See d7fee20